client/ayon_harmony/api/pipeline.py

```diff
@@ -252,14 +252,14 @@ def ls():
         dict: container
     """
     scene_data = harmony.get_scene_data() or dict()
-    all_top_names = harmony.get_all_top_names()
+    containers_names = harmony.get_all_top_names() | harmony.get_palettes_paths()
     cleaned_scene_data = True
```

Comment:
Is it correct that this is **always** True?

Reply:
You're right, it's a mistake, I'll fix it.